.travis.yml

```diff
@@ -28,8 +28,6 @@ jobs:
     - name: "Ubuntu: no configure flags"
       stage: test
       compiler: gcc
-      env:
-       - PYTHON_VERSION=2.7
     - name: "Ubuntu: py3.6 distcheck"
```

Comment:
Maybe it is worth having some tests run explicitly against 3.6 and some against 3.7?  I'm not sure what the various distros have by default for python3.

Reply:
Yeah, getting 3.8 in there and maybe 3.7 wouldn't be a bad idea.  We definitely have 3.7.2 in TCE on TOSS, so that would be a good one to keep tabs on, and since 3.8 has been out long enough we could add that one too.

Reply:
I'm building a new bionic docker image for this right now, we'll have 3.7 and 3.8 in travis shortly.